app/components/certificate/certificate-available.tsx

```diff
@@ -1,4 +1,4 @@
-import { Divider } from '@chakra-ui/react';
+import { Divider, Spacer } from '@chakra-ui/react';
```

Comment:
> This is just my opinion. To me, it's a bit weird that `Created On` is not left-aligned while other contents are all left-aligned.

It's more for the way the dates are aligned, because we are not displaying each date as a row, it's a bit odd for me to not make them centered, but if you have a different approach I'll be glad to take a look and change it